### PR TITLE
Adding a new test case for test_auto_reclaim_space_cronjob.py test

### PR DESCRIPTION
**Pull request by ypersky1980**
In this PR I've added a test case 'test_reclaim_space_cronjob_on_existing_namespace' which verifies the creation and deletion of cronjobs upon pvc creation/deletion in the existing "openshift-monitoring" namespace. 

This test case has a lot of common code with already existing test case 'test_reclaim_space_cronjob' . 
Therefore the shared code was refactored into  __reclaim_space_cronjob method.

### CODE REVIEW

**Review by shivamdurgbuns — DISMISSED** [review]

File: tests/manage/pv_services/space_reclaim/test_auto_reclaim_space_cronjob.py [hunk 1]
```diff
@@ -220,9 +255,12 @@ def wait_for_cronjobs(self, cronjobs_exist, msg, timeout=60):
                 cmd="get reclaimspacecronjob",
                 namespace=self.namespace,
             ):
-                if (len(sample) > 1 and cronjobs_exist) or (
-                    len(sample) == 1 and not cronjobs_exist
-                ):
+                # if (len(sample) > 1 and cronjobs_exist) or (
+                #     len(sample) == 1 and not cronjobs_exist
+                # ):
```
Comment (shivamdurgbuns) [inline tests/manage/pv_services/space_reclaim/test_auto_reclaim_space_cronjob.py:260]: we can delete this commented line if they are not needed.

Comment (ypersky1980) [reply tests/manage/pv_services/space_reclaim/test_auto_reclaim_space_cronjob.py]: Fixed. Thank you!

**Review by keesturam — APPROVED** [review]
LGTM

**Review by pintojoy — APPROVED** [review]
LGTM

**Pull request merged**